src/xcode/ENA/ENA/Source/Scenes/Checkin/QRCodeScanner/CheckinQRCodeScannerViewController.xib

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="17701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
```

Comment:
Why copy this XIB from the 'older' QRCode Scanner view?
It would ne much better to finish the 'new' one with out that xib file.

Reply:
Just not to re-invent the wheel, the other way is too add the mask layer and the label by code,but if you prefer this i can do as well.

Reply:
the qr code scanner code got copied for tjhe reason to make thing better this time
Please trie to add  the layer by code.